app/src/main/java/com/example/waifu/MainActivity.kt

```diff
@@ -7,12 +7,20 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.BaseAdapter
 import android.widget.ListView
+import androidx.core.text.set
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        var taskGoalName = editTextName.text
+
+        button_confirm.setOnClickListener {
+            editTextName.text.clear()
```

Comment:
Added storing the text from the editText box allowing to store the name of your goal/task as well as clearing out the box when the button is clicked

Reply:
Is this used anywhere?  If not, it's an incomplete feature.